Source/WebKit/Shared/LayerTreeContext.h

```diff
@@ -45,11 +40,9 @@ enum class LayerHostingMode : uint8_t {
 class LayerTreeContext {
 public:
     LayerTreeContext();
```

Comment:
This constructor now has uninitialized memory.  Can it be removed?  If both are removed, it should just compile.